src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -405,7 +408,10 @@ const saveMetacard = async (parent, args, context) => {
       id,
       'metacard.modified': modified,
       'metacard.owner': newMetacardAttrs.metacard_owner,
-      ...attributes,
+      ...res.updatedMetacards[0].attributes,
```

Comment:
should we be returning all the attributes in a metacard?

Reply:
The graphql only returns the attributes that are asked for in the `gql`, so it won't actually end up sending all attributes over the network